applications/nrf_desktop/doc/battery_meas.rst

```diff
@@ -0,0 +1,60 @@
+.. _battery_meas:
+
+Battery measurement
+###################
+
+The ``battery_meas`` module periodically measures the battery voltage and sends ``battery_level_event`` informing about the current battery level.
+
+Module Events
+*************
+
++----------------+-------------+------------------+-------------------------+------------------+
+| Source Module  | Input Event | This Module      | Output Event            | Sink Module      |
++================+=============+==================+=========================+==================+
+|                |             | ``battery_meas`` | ``battery_level_event`` | :ref:`bas`       |
++----------------+-------------+------------------+-------------------------+------------------+
+
+Configuration
+*************
+
+The module uses the Zephyr :ref:`zephyr:adc` driver to measure voltage.
+For this reason, you should set the following options:
+
+* :option:`CONFIG_ADC`
+* :option:`CONFIG_ADC_ASYNC` - implementation uses asynchronous calls.
+* :option:`CONFIG_ADC_0` - implementation uses ``ADC_0``.
+* :option:`CONFIG_ADC_NRFX_SAADC` - implementation uses nrfx SAADC driver for nRF52 MCU series.
+
+Set ``CONFIG_DESKTOP_BATTERY_MEAS`` to enable the module.
+Define ``DESKTOP_BATTERY_MEAS_POLL_INTERVAL_MS`` - time between subsequent battery measurements (in ms).
+
+Set ``CONFIG_DESKTOP_BATTERY_MEAS_HAS_VOLTAGE_DIVIDER`` option if measurement circuit contains voltage divider made of two resistors.
+
+Values of mentioned resistors (in kOhm) are defined as:
+
+* ``CONFIG_DESKTOP_BATTERY_MEAS_VOLTAGE_DIVIDER_UPPER`` - upper resistor.
+* ``CONFIG_DESKTOP_BATTERY_MEAS_VOLTAGE_DIVIDER_LOWER`` - lower resistor.
+
+The module measures the voltage over the lower resistor.
+
+Battery voltage is converted to state of charge (SoC) according to lookup table defined in ``battery_def.h`` file located in board-specific directory in the application configuration folder.
+
+You should also define battery parameters (default values will be used if options are left undefined):
+
+* ``CONFIG_DESKTOP_BATTERY_MEAS_MIN_LEVEL`` - battery voltage that corresponds to 0% level (in mV).
+* ``CONFIG_DESKTOP_BATTERY_MEAS_MAX_LEVEL`` - battery voltage that corresponds to 100% level (in mV).
+* ``CONFIG_DESKTOP_VOLTAGE_TO_SOC_DELTA`` - difference between adjacent elements in conversion LUT (in mV).
+
+If a pin is used to enable battery measurement, option ``CONFIG_DESKTOP_BATTERY_MEAS_HAS_ENABLE_PIN`` must be enabled.
+Selected pin number must be defined as ``CONFIG_DESKTOP_BATTERY_MEAS_ENABLE_PIN`` (implementation uses ``GPIO0`` port).
```

Comment:
```suggestion
The number of the pin used for this purpose must be defined as ``CONFIG_DESKTOP_BATTERY_MEAS_ENABLE_PIN``, because the implementation uses the ``GPIO0`` port.
```

Reply:
It is more like implementation always uses GPIO0 port, bout you can select the pin. I will reword this.